docs/data/toolpad/core/components/account/account.md

```diff
@@ -30,16 +30,45 @@ When signed out, the component renders as an inline sign in button within the da
 
 ## Customization
 
+The `<Account />` component is composed of multiple sub-components:
+
+- `<SignInButton />`
+- `<AccountPreview />`
+- `<Popover />`
+- `<SignOutButton />`
+
+You can pass extra props to them through the `slotProps` prop on the `<Account />` component. You may also completely override them by passing your own custom components to `<Account />` through the `slots` prop.
+
+The labels can be customized through the `localeText` prop.
+
 ### Slot Props
 
-The underlying `signInButton`, `signOutButton` and `iconButton` components can be customized by passing in `slotProps` to the `Account` component.
+The `AccountPreview` component has two variants, `condensed` (the default) and `expanded`. You can change the variant used inside `<Account />` by passing in custom props through `slotProps`:
 
-Labels for the sign in and sign out buttons can be customized through the `localeText` prop.
+{{"demo": "AccountCustomSlotProps.js", "bg": "outlined" }}
 
-{{"demo": "AccountCustom.js", "bg": "outlined" }}
+### Labels
```

Comment:
Maybe we can order sections like "slot props", "slots" and only then the "Labels"? Feels more logical to have the slot sections together, maybe.